mobile/src/libs/hooks/use-search/constants.ts

```diff
@@ -0,0 +1,5 @@
+const DEFAULT_SEARCH_PAYLOAD = {
+  search: '',
+};
+
+export { DEFAULT_SEARCH_PAYLOAD };
```

Comment:
why you didn't create the 'libs' folder?

Reply:
Done